src/Sarif.Driver/Sdk/EntryPointUtilities.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Sarif.Driver.Sdk
+{
+    public static class EntryPointUtilities
+    {
+        internal static string[] GenerateArguments(
+            string[] args,
+            IFileSystem fileSystem,
+            IEnvironmentVariables environmentVariables)
+        {
+            List<string> expandedArguments = new List<string>();
+
+            foreach (string argument in args)
+            {
+                if (argument[0] != '@' &&
+                    (argument.Length == 1 || argument[1] != '@'))
+                {
+                    expandedArguments.Add(argument);
+                    continue;
+                }
+
+                string responseFile = argument.Trim('"').Substring(1);
+                try
+                {
+                    responseFile = environmentVariables.ExpandEnvironmentVariables(responseFile);
+                    responseFile = fileSystem.GetFullPath(responseFile);
+
+                    if (!fileSystem.FileExists(responseFile))
+                        responseFile = null;
```

Comment:
>; [](start = 43, length = 1)

See my comment in the unit tests file (which I reviewed first, that's why the comment is there). I think this is an error condition.